code-name-taran/src/com/spaghettisoft/component/sixtysyx/SixtySix.java

```diff
@@ -0,0 +1,268 @@
+package com.spaghettisoft.component.sixtysyx;
+
+import com.spaghettisoft.component.game.SixtySixAbstractGame;
+import com.spaghettisoft.component.sixtysyx.resourses.Deck;
+import com.spaghettisoft.component.sixtysyx.resourses.PlayingCard;
+import com.spaghettisoft.globals.StaticObjects;
+
+public class SixtySix extends SixtySixAbstractGame
+{
+	private Player playerOne;
+	private Player playerTwo;
+	private Deck deck;
+	public static PlayingCard powerCard;
+	private boolean isGameClosed;
+	private boolean isFirstPlayerHandWinner;
+	private boolean isFirstHand;
+
+	public SixtySix()
+	{
+		isGameClosed = false;
+		isFirstPlayerHandWinner = true;
+		isFirstHand = true;
+	}
+
+	@Override
+	protected void initialize()
+	{
+		System.out.println("============================");
+		getPlayerNames();
+		deck = new Deck();
+		dealCards();
+		drawPowerCard();
+		System.out.println("============================");
+	}
+
+	private void getPlayerNames()
+	{
+		showTitle();
+		System.out.print("Player One Name: ");
+		String firstPlayerName = StaticObjects.scanner.nextLine();
+		System.out.print("Player Two Name: ");
+		String secondPlayerName = StaticObjects.scanner.nextLine();
+
+		playerOne = new Player(firstPlayerName);
+		playerTwo = new Player(secondPlayerName);
+	}
+
+	private void dealCards()
+	{
+		deal();
+		deal();
+		playerOne.orderHand();
+		playerTwo.orderHand();
+	}
+
+	private void deal()

```

Comment:
You can simplify the method by making it just to return you 3 cards from the deck. Then you can call it 4 times and use the return values to put the cards in each player's hand.